adbutils/_utils.py

```diff
@@ -18,6 +20,13 @@
 
 MB = 1024 * 1024
 
+
+def append_path(base: typing.Union[str, pathlib.Path], addition: str) -> str:
+    if isinstance(base, pathlib.Path):
+        return str(base / addition)
```

Comment:
in windows, the path use "\\" as seperator, which may not working in android shell. it's bettery use as_posix() instead of str

Reply:
Done in [f9612d2](https://github.com/openatx/adbutils/pull/100/commits/f9612d2b416541b67ea95000d3db281b0676dedf)